src/subclass/list_box_row.rs

```diff
@@ -0,0 +1,14 @@
+use glib::subclass::prelude::*;
```

Comment:
Also, please don't use wildcard import in the crate directly.

Reply:
Why not? We do that elsewhere too :)

Reply:
We shouldn't. My thinking here is that it makes the code harder (and eventually slower to compile) in the location where it shouldn't. The crate provides the API, so we should make clear what it uses. I can update the other places if needed.

Reply:
We're not in `glib` here but in `gtk`, so it comes from another crate :) And it doesn't change anything with regards to compile times: the compiler knows all the traits anyway, or how would it otherwise tell you "maybe you want to put this trait into scope for being able to call this function"?

By not including the prelude from GLib here we will need more inter-crate changes whenever some trait inside glib changes its name or functions get moved between traits or traits get split or ...

Reply:
So I completely disagree here, changing it to explicit imports only causes more inter-crate dependencies and we really don't want that. Too bad @kavanmevada already changed it now...